h2o-bindings/bin/custom/python/gen_pca.py

```diff
@@ -17,3 +28,149 @@ def init_for_pipeline(self):
 extensions = dict(
     __class__=class_extensions,
 )
+
+examples = dict(
+    compute_metrics="""
+>>> prostate = h2o.import_file("http://s3.amazonaws.com/h2o-public-test-data/smalldata/prostate/prostate.csv.zip")
+>>> prostate['CAPSULE'] = prostate['CAPSULE'].asfactor()
+>>> prostate['RACE'] = prostate['RACE'].asfactor()
+>>> prostate['DCAPS'] = prostate['DCAPS'].asfactor()
+>>> prostate['DPROS'] = prostate['DPROS'].asfactor()
+>>> pros_pca = H2OPrincipalComponentAnalysisEstimator(compute_metrics=False)
+>>> pros_pca.train(x=prostate.names, training_frame=prostate)
+>>> pros_pca.show()
+""",
+    export_checkpoints_dir="""
+>>> import tempfile
+>>> from os import listdir
+>>> prostate = h2o.import_file("http://s3.amazonaws.com/h2o-public-test-data/smalldata/prostate/prostate.csv.zip")
+>>> prostate['CAPSULE'] = prostate['CAPSULE'].asfactor()
+>>> prostate['RACE'] = prostate['RACE'].asfactor()
+>>> prostate['DCAPS'] = prostate['DCAPS'].asfactor()
+>>> prostate['DPROS'] = prostate['DPROS'].asfactor()
+>>> checkpoints_dir = tempfile.mkdtemp()
+>>> pros_pca = H2OPrincipalComponentAnalysisEstimator()
```

Comment:
Need to specify export_checkpoints_dir here. 
pros_pca = H2OPrincipalComponentAnalysisEstimator(export_checkpoints_dir=checkpoints_dir)

Reply:
Fixed.